packages/avatar/src/react/index.tsx

```diff
@@ -44,60 +44,61 @@ const styles: { [key: string]: StyleFn } = {
   initials: () => glamor.css(stylesheet['.psds-avatar__initials'])
 }
 
-const Avatar = React.forwardRef<HTMLDivElement, AvatarProps>((props, ref) => {
-  const { alt, name, size: _size, src, ...rest } = props
-
-  const [imageState, setImageState] = React.useState<ImageState>('loading')
-
-  const handleImageLoadSuccess: React.ReactEventHandler<HTMLImageElement> = evt => {
-    const pixel = evt.target as HTMLImageElement
-    const isFallbackPixel = pixel.naturalWidth === 1
-    setImageState(isFallbackPixel ? 'error' : 'success')
-  }
-
-  const handleImageLoadError: React.ReactEventHandler<HTMLImageElement> = () => {
-    setImageState('error')
+export const Avatar = React.forwardRef<HTMLDivElement, AvatarProps>(
```

Comment:
Is there anything of substance in this file diff?

Reply:
Named export now